models/ed/R/met2model.ED2.R

```diff
@@ -114,14 +114,25 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
     sec  <- nc$dim$time$vals
     Tair <- ncdf4::ncvar_get(nc, "air_temperature")
     Qair <- ncdf4::ncvar_get(nc, "specific_humidity")  #humidity (kg/kg)
-    U    <- ncdf4::ncvar_get(nc, "eastward_wind")
-    V    <- ncdf4::ncvar_get(nc, "northward_wind")
+    U    <- try(ncdf4::ncvar_get(nc, "eastward_wind"),  silent = TRUE)
+    V    <- try(ncdf4::ncvar_get(nc, "northward_wind"), silent = TRUE)
     Rain <- ncdf4::ncvar_get(nc, "precipitation_flux")
     pres <- ncdf4::ncvar_get(nc, "air_pressure")
     SW   <- ncdf4::ncvar_get(nc, "surface_downwelling_shortwave_flux_in_air")
     LW   <- ncdf4::ncvar_get(nc, "surface_downwelling_longwave_flux_in_air")
     CO2  <- try(ncdf4::ncvar_get(nc, "mole_fraction_of_carbon_dioxide_in_air"), silent = TRUE)
 
+    use_UV <- is.numeric(U) & !is.numeric(V)
```

Comment:
does it make sense to first check and see if wind_speed is available and use that, if not available use U and V, and print error if neither is available.

Reply:
yeah it does, the thing is we (almost always?) write `U` and `V` in the standard CFs, PalEON data is a special case. I'll revise this also after I make sure Mike is OK with this implementation overall

Reply:
This is not what intended. The CF standard is to write U & V. ED2 wants U & V. If U & V are absent then we can approximate U = wind speed and V = 0, but we definitely shouldn't be doing that by default.

Really confused by the logic of is.numeric(U) & !is.numeric(V) and when that would apply -- seems like we're either going to have both or we'll have neither, but I can't imagine why we'd have U but not V and how we could calculate sqrt(U ^ 2 + V ^ 2) if we don't have V. 

If we have both then use them. If we have neither check for wind_speed.

Reply:
one with `!` one without was definitely a typo

OK I'm going to change it back to writing both U and V